### PR TITLE
[chore](recover) add a config to recover remaining data in emergency

### PR DESCRIPTION
**Pull request by yangzhg**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yangzhg** [comment]
run allbuild

**Comment by yangzhg** [comment]
run buildall

**Comment by yangzhg** [comment]
run buildall

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -1439,6 +1439,20 @@ public class Config extends ConfigBase {
...
+     * necessary to restore the remaining data (which may cause problems with the correctness of the data).
+     * This configuration is the same as` recover_with_empty_tablet` should only be used in emergency situations
+     * This configuration has three values:
+     *   disable : If an exception occurs, an error will be reported normally.
+     *   ignore_version: ignore the visibleVersion information recorded in fe partition, use replica version
+     *   ignore_all: In addition to ignore_version, when encountering no queryable replica,
+     *   skip it directly instead of throwing an exception
+     */
+    @ConfField(mutable = true, masterOnly = true)
+    public static String revover_with_skip_missing_version = "disable";
```
Comment (yongjinhou) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:1454]: revover -> recover ???

**Comment by yangzhg** [comment]
run buildall

**Review by adonis0147 — APPROVED** [review]
LGTM

**Pull request merged**